src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -132,7 +134,10 @@ class OPENVINO_GENAI_EXPORTS GenerationConfig {
     bool is_greedy_decoding() const;
     bool is_beam_search() const;
     bool is_multinomial() const;
+    OPENVINO_DEPRECATED("Please, specify device explicitly when create LLMPipeline. This overload will be removed in 2025.0.0 release")
```

Comment:
a deprecated notice comment is copy-paste. You need to insert a text dedicated to your API